README.md

```diff
@@ -376,23 +427,18 @@ You may need to stash changes if you have modified the `docker-compose.yml` file
 
 Similarly, if `prover_config.json` or `dry_run_config.json` have been modified, ensure the changes are applied again.
 
-### Delete Volume
+### Run Upgrade Script
```

Comment:
Not sure whether we need mentioned "As we changed to use our custom mongodb image with extra data at volume mongodb_data, so we do not need the legacy mapped mongo directory.
So if you want to save your disk space you can remove the mongo directory under this repo's dir."